### PR TITLE
system76-power: Add at v1.2.1

### PR DESCRIPTION
**Pull request by Pheoxy**
https://github.com/serpent-os/recipes/issues/353

### CODE REVIEW

File: s/system76-power/stone.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+version     : 1.2.1
+release     : 1
+homepage    : https://github.com/pop-os/system76-power
+upstreams   :
+    - https://github.com/pop-os/system76-power/archive/refs/tags/1.2.1.tar.gz: 90b796e72a442a7968fc80b269ed2a936c6541bcba939a72612898082fbd7ec3
+summary     : system76-power is a utility for managing graphics and power profiles.
+description : |
+    system76-power is a utility for managing graphics and power profiles.
+license:
+    - GPL-3.0
```
Comment (joebonrichie) [inline s/system76-power/stone.yaml:16]: nit: use a valid spdx licence identifier. Looks like it's `GPL-3.0-only` in this case https://github.com/pop-os/system76-power/blob/master/src/acpi_platform.rs#L2

File: s/system76-power/stone.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    - binary(cargo)
+    - pkgconfig(libusb-1.0)
+rundeps     :
+    - pkgconfig(dbus-1)
+    - pkgconfig(libusb-1.0)
+networking  : yes
+build       : |
+    %make
+install     : |
+    %make sysconfdir="/usr/lib" DESTDIR="%(installroot)" install
```
Comment (joebonrichie) [inline s/system76-power/stone.yaml:27]: nit: there is a `%make_install` macro

File: s/system76-power/stone.yaml (hunk 1 above)
Comment (Pheoxy) [reply s/system76-power/stone.yaml:16]: Done and thanks I've added a this to #https://github.com/serpent-os/serpentos.com/issues/20 as well.

File: s/system76-power/stone.yaml (hunk 2 above)
Comment (Pheoxy) [reply s/system76-power/stone.yaml]: Changed to `%make_install`

**Comment by Pheoxy** [comment]
Probably should have done another commit for reviewed changes...

**Review by joebonrichie — APPROVED** [review]

**Pull request merged**